### PR TITLE
Dear robots and humans, please read this

### PR DESCRIPTION
**Pull request by hikes101**
Added robots.txt and humans.txt according to [cloudflare](https://www.cloudflare.com/learning/bots/what-is-robots.txt/)

### CODE REVIEW

**Comment by hikes101** [comment]
[Pending: Create robots.txt ?](https://trello.com/c/4EPHQ2OH/162-pending-create-robotstxt)

File: robots.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+User-agent: *
+Disallow: /blog/create
+Crawl-delay: 8
```
Comment (pheeria) [inline robots.txt]: Why a delay here?

Comment (hikes101) [reply robots.txt]: "Crawl-delay: The crawl delay command is meant to stop search engine spider bots from overtaxing a server. It allows administrators to specify how long the bot should wait between each request, in milliseconds."

Thought it doesn't hurt but we also don't really need it. I can remove it if you want.

Comment (pheeria) [reply robots.txt]: Please remove it :)

**Comment by pheeria** [comment]
Sorry, I forgot to mention that the files should be under `/static` directory.
They should be available locally under `localhost:3001/robots.txt`, which they are currently most probably not.

**Pull request merged**